scripts/utils.py

```diff
@@ -23,14 +23,11 @@
 # *  e-mail address 'scipion@cnb.csic.es'
 # ***************************************************************************/
 
-import subprocess
+import subprocess, glob, distutils.spawn
```

Comment:
We prefer each import in a line, easy to read

Reply:
okay, changing